### PR TITLE
Fix multi-child fragment

### PR DESCRIPTION
**Pull request by jchavarri**
Currently, this code `<> <div/> <div/> </>` will trigger the infamous warning: `Each child in a list should have a unique "key" prop`.

This PR modifies the expression switch for fragments so that those with 0 or 1 child use `jsx` and those with 2 or more children use `jsxs`.

### CODE REVIEW

**Review by davesnx — APPROVED** [review]
Nice fix

File: .ocamlformat [hunk 1]
```diff
@@ -1 +1 @@
-version = 0.26.0
+version = 0.26.2
```
Comment (tatchi) [inline .ocamlformat:1]: Is this change on purpose or because you had this version locally in your switch? IOW, should it be commited?

**Comment by tatchi** [comment]
Nice, thanks! 🤗

File: .ocamlformat (hunk 1 above)
Comment (jchavarri) [reply .ocamlformat]: You're right, reverted in https://github.com/reasonml/reason-react/pull/852/commits/e5eeb3f23fd08fe028f7220f87caa5b86d3161c2.

**Review by anmonteiro — APPROVED** [review]
I pushed a small refactor that avoids sharing the code in the entire branch (just decides between `jsx` and `jsxs`)
